chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+is always stored in the most efficient format.
+
+% Documentation
+Throughout the data cleaning process, you will need inputs from the field, 
+including enumerator manuals, survey instruments, 
+supervisor notes, and data quality monitoring reports.
+These materials are part of what we call data documentation
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Documentation}}
+\index{Documentation},
+and should be stored in the corresponding folder, 
+as you will probably need them during analysis and publication.
+Include in the \texttt{Documentation} folder records of any
+corrections made to the data, including to duplicated entries,
+as well as communications from the field where theses issues are reported.
+Make sure to also have a record of potentially problematic patterns you noticed
+while exploring the data, such as outliers and variables with many missing values.
+Be very careful not to include sensitive information in 
+documentation that is not securely stored, 
+or that you intend to release as part of a replication package or data publication.
+
+\section{Indicator construction}
 
+% What is construction -------------------------------------
+Any changes to the original data set happen during construction.
+It is at this stage that the raw data is transformed into analysis data.
+This is done by creating derived variables
+(binaries, indices, and interactions, to name a few).
+To understand why construction is necessary,
+let's take the example of a household survey's consumption module.
+It will result in separate variables indicating the 
+amount of each item in the bundle that was consumed.
+There may be variables indicating the cost of these items.
+You cannot run a meaningful regression on these variables. 
+You need to manipulate them into something that has \textit{economic} meaning. 
+During this process, the data points will typically be reshaped and aggregated 
+so that level of the data set goes from the unit of observation in the survey to the unit of analysis.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Unit\_of\_Observation}} 
+To use the same example, the data on quantity consumed was collect for each item, and needs to be aggregated to the household level before analysis.
+
+% Why it is a separate process -------------------------------
+
+% From cleaning
+Construction is done separately from data cleaning for two reasons. 
+The first one is to clear differentiation between the data originally collected and the result of data processing decisions.
+The second is to ensure that variable definition is consistent across data sources. 
+Unlike cleaning, construction can create many outputs from many inputs. 
+Let's take the example of a project that has a baseline and an endline survey. 
+Unless the two instruments are exactly the same, which is unlikely,  the data cleaning for them will require different steps, and therefore will be done separately. 
```

Comment:
_, which is unlikely,_ - it sounds as if we are indifferent to whether it is or not. how about:

_, which is preferably but often not the case,_

Reply:
>Unless the two instruments are exactly the same, which is preferable but often not the case,  the data cleaning for them will require different steps, and therefore will be done separately.

(c80a095)

Reply:
Even if the instruments are the same, data cleaning will be done at different times and should be different - the same errors are not going to arise even if exactly the same instrument is fielded twice, and there's also sample changes to consider (e.g. attrition)